src/knex-processor.ts

```diff
@@ -22,7 +22,7 @@ export default class KnexProcessor<
       .select();
   }
 
-  async remove(data: Resource): Promise<boolean> {
+  async remove(data: Resource): Promise<null> {
```

Comment:
@Exelord I think it's more semantically correct to return a `boolean` here. However, if you mean to expect no return value, it's probably better to use `Promise<void>`.

Reply:
I was thinking about that, but as `remove` is a promise there is no need for any response. The promise will resolve or not. There won't be any chance to get other response than `true` (1) so it doesn't make sense to use it. Would you agree?